downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -97,6 +97,44 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%222201.9.0%22+label%3AType%2FBug).
 
+## Ballerina extended library updates
```

Comment:
As per the offline discussion we (@shafreenAnfar, @daneshk, @ThisaruGuruge and I) had, it was decided to list down connector release notes under a separate section to improve the clarity.